orcid-persistence/src/main/resources/db/updates/1.0.19.xml

```diff
@@ -1252,7 +1252,7 @@
         <insert tableName="country_reference_data">
 
             <column name="country_name"
```

Comment:
@DanielPalafox please rollback this and create a new script to update the name of the country in the DB.

Thanks

Reply:
@amontenegro done!